tests/testthat/test_pairwise.R

```diff
@@ -163,6 +163,20 @@ test_that("test do_dist with cmd_scale", {
   expect_equal(result_kv[[4]], result_kv[[4]])
```

Comment:
This one should be updated too. And it should have some comments on the expected values like 'this is normalized', 'this is not normalized'.

Reply:
Added expected value.
About normalized or not, first argument of test_that() indicates that.
```
test_that("test do_dist with cmd_scale with normalize", {
```